File: COMPILADOR/kokoslan_/src/src/java/ast/KoKoId.java
/**
 @author loriacarlos@gmail.com
 
*/
package src.java.ast;

import src.java.eval.KoKoContext;
import src.java.eval.KoKoValue;


public class KoKoId extends KoKoAtom<String>{
   public KoKoId(String value){
      super(value);
   }
   @Override 
   public KoKoValue eval(KoKoContext ctx){
	   return ctx.find(this);
   }
   
}

File: COMPILADOR/kokoslan/src/KoKoKotlin/ast/KoKoId.kt
/**
 * @author loriacarlos@gmail.com
 */
package KoKoKotlin.ast

import KoKoKotlin.eval.KoKoContext
import KoKoKotlin.eval.KoKoValue


class KoKoId(value: String) : KoKoKotlin.ast.KoKoAtom<String>(value) {
    override fun eval(ctx: KoKoContext): KoKoValue {
        return ctx.find(this)
    }

}